sdk/data_cosmos/src/clients/document.rs

```diff
@@ -42,6 +42,26 @@ impl DocumentClient {
         ReplaceDocumentBuilder::new(self.clone(), document)
     }
 
+    /// Patch the document.
+    /// The Patch Document operation does path-level updates to specific files/properties in a single document.
+    ///
+    /// # Example
+    /// ```no_run
```

Comment:
```suggestion
    /// ```no_run
    /// # use azure_data_cosmos::prelude::*;
    /// # async fn func(document_client: &DocumentClient) -> azure_core::Result<()> {
```